edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/edgedeployment/PodExtensions.cs

```diff
@@ -43,31 +43,35 @@ static Option<V1ContainerStatus> GetContainerByName(string name, V1Pod pod)
             return Option.Maybe(status);
         }
 
-        static ReportedModuleStatus ConvertPodStatusToModuleStatus(Option<V1ContainerStatus> podStatus)
+        static ReportedModuleStatus ConvertPodStatusToModuleStatus(V1PodStatus podStatus)
         {
-            return podStatus.Map(
-                pod =>
+            ModuleStatus moduleStatus = ModuleStatus.Failed;
```

Comment:
in which cases this default values will be used? now it is a little bit confusing